src/StateResolver/OrderTransitions.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\RefundPlugin\StateResolver;
+
+final class OrderTransitions
+{
+    public const GRAPH = 'sylius_order';
+
+    public const TRANSITION_REFUND = 'refund';
```

Comment:
Even though in **Sylius** this and `OrderStates` class are separated, I think in plugin these constants could be placed in one class